tools/mason/MasonUpdate.chpl

```diff
@@ -41,38 +41,56 @@ private var failedChapelVersion : [1..0] string;
 
 /* Finds a Mason.toml file and updates the Mason.lock
    generating one if it doesnt exist */
-proc UpdateLock(args: [] string, tf="Mason.toml", lf="Mason.lock") {
-  if isFile(tf) {
+proc UpdateLock(args: [] string, tf="Mason.toml", lf="Mason.lock", isTest=false) {
```

Comment:
I'd prefer to keep testing logic out of the interface (even though it's not user-facing). I just sent you a patch for the mason tests that I think will eliminate the need for this argument.

Reply:
applied